### PR TITLE
Chore: Reduce pa11y-ci errors

### PR DESCRIPTION
**Pull request by hugohaggmark**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: packages/grafana-e2e-selectors/src/selectors/components.ts [hunk 1]
```diff
@@ -72,7 +72,11 @@ export const Components = {
         },
       },
       BarGauge: {
+        /**
+         * @deprecated use valueV2 from Grafana 8.3 instead
+         */
         value: 'Bar gauge value',
+        valueV2: 'data-testid Bar gauge value',
```
Comment (axelavargas) [inline packages/grafana-e2e-selectors/src/selectors/components.ts:79]: Aha 😃 , this is the new way to keep our selectors backward compatible right? Is it maybe a good idea to add this into the [styleguide](https://github.com/grafana/grafana/blob/main/contribute/style-guides/e2e.md)?

**Review by axelavargas — COMMENTED** [review]
I know still in draft, but took a look, nice job 🙌🏾 , I think we also need to reduce the thresholds in Pa11y, I saw a nice 0 on the gauge visualization.

File: packages/grafana-e2e-selectors/src/selectors/components.ts (hunk 1 above)
Comment (hugohaggmark) [reply packages/grafana-e2e-selectors/src/selectors/components.ts:79]: You're correct, if you have any feedback or suggestions I'm really appreciate it. I'd like to see/receive feedback before adding it to the styleguide though, but I totally agree thats where it belongs 👍

**Comment by hugohaggmark** [comment]
@axelavargas I lowered all the thresholds in my last commit, I wasn't sure I would make it to the end of the list before the end of today so I left that as a last thing todo. Would appreciate an approval before the release tomorrow morning. Thank you 🙏

File: packages/grafana-e2e-selectors/src/selectors/components.ts (hunk 1 above)
Comment (axelavargas) [reply packages/grafana-e2e-selectors/src/selectors/components.ts:79]: I like the idea of versioning our selectors and using the deprecated annotation to be explicit is very useful, it follows the same pattern we use in some places of the protect :)

**Review by axelavargas — APPROVED** [review]
Great job in this PR 🚀 @hugohaggmark

**Pull request merged**